Makefile

```diff
@@ -140,5 +140,5 @@ $(GOVENDOR):
 .PHONY: testdeps
 testdeps: $(GOVENDOR)
 	$(GOVENDOR) install +local
-	# Fail if gometalinter is not present in $PATH:
+	# Fail if gometalinter is not present in PATH:
```

Comment:
Wanna break this out into another PR / commit?

Reply:
They're already separate commits, but I can break them into another PR, sure.